SA3/qa_web/tests/test_models.py

```diff
@@ -1,20 +1,45 @@
 from django.test import TestCase
 from django.contrib.auth import authenticate
-from qa_web.models import Questions, Comments, User
+from qa_web.models import Questions, Comments, Answers, User, Vote
 
+import random
 
-class ModelTest(TestCase):
+credentials = {'username' : 'johnny', 'password' :'password123'}
 
-    def test_comments(self):
-        uname = 'johnny'
-        pw = 'password123'
-        user1 = authenticate(username=uname, password=pw)
+def obtain_sample_objects_as_tuple():
 
+    user1 = User.objects.first()
+    q = Questions.objects.first()
+    a = Answers.objects.first()
+    return user1, q, a
+
+def create_many_users():
+
+    for i in range(0,15):
+        name = 'User{}'.format(i)
+        pw = '{}'.format(i)
+        u = User.objects.create(username = name, password= pw)
+    return
+
+class CommentModel(TestCase):
+
+    def setUp(self):
+        User.objects.create_user(**credentials)
+        user1 = User.objects.first()
         title = "What is life?"
         content = "Someone please explain to me the purpose of life"
         q = Questions.objects.create(title=title, content=content, owner=user1)
+        a = Answers.objects.create(content= 'Test answer', owner = user1, correct_answer= True)
+
+    def test_to_string_method_question(self):
+        _, q, _ = obtain_sample_objects_as_tuple()
```

Comment:
Since I created `obtain_sample_objects_as_tuple()` as a helper function for fetching database objects, I wanted to be consistent with its use throughout every test cases. Its use is then more logical in my opinion.

Reply:
I agree. You could keep your style and and others will understand your logic if there are lots test cases. And i think it might be better that we could add more comments on some testing methods so people will see author's intention.